src/Paths.cs

```diff
@@ -2,7 +2,7 @@ namespace Silksong.DataManager;
 
 using IO = System.IO;
 
-public static class DataPaths
+public static class Paths
```

Comment:
Keep in mind this name is going to appear unqualified in most client code; DataPaths is a lot more specific than Paths (which paths? one might ask).